### ISSUE TITLE
Is the workload_identity_provider or the service_account a secret?

### ISSUE THREAD

**Issue by TWiStErRob** [google-github-actions/auth#244]
### TL;DR

Update readme.md answering: Is Workload Identity Provider resource name considered a secret?

### Detailed design

Please update #setup in README.md
https://github.com/google-github-actions/auth/blob/main/README.md#setup
and explicitly state whether or not it is recommended to store the `WORKLOAD_IDENTITY_POOL_ID` generated in Step 6 and/or the Workload Identity Provider resource name value generated at Step 9 as a GitHub secret.

### Additional information

Please also update https://github.com/google-github-actions/auth/blob/main/action.yml#L28.

And if it is a secret, please replace the example usages of `workload_identity_provider:` accordingly.

**Comment by sethvargo** [comment]
Hi @TWiStErRob - there's no "right" answer here, as it depends on your threat model and repository setup. WIF pools, WIF providers, and service account email addresses are not secrets like private keys or passwords, but then _can_ reveal information depending on your naming scheme (similar to project IDs). Knowing the WIF pool or provider does not grant access to it, so it's really dependent on your naming scheme and threat model.

Furthermore, there are reasons why you might use GitHub Secrets for non-secret values. For example, if you provisioned all your repositories using an IaC tool like Terraform, you might want to inject values into the GitHub repo. The easiest way to do that is via secrets, even if the values themselves are not "secret". I hope that helps.

**Comment by TWiStErRob** [comment]
Make sense, thanks. Could you please add this in some form to the readme?

**Comment by jxc876** [comment]
I was wondering the same thing, wether I should be considering these are secrets or if they could be regular variables.
Thanks for the answer.
